Auto-Quartett_WindowsForms/Editor.cs

```diff
@@ -16,5 +16,51 @@ public Editor()
         {
             InitializeComponent();
         }
+        Autokarte eintrag = new Autokarte();
+        private void btnSpeichern_Click(object sender, EventArgs e)
+        {
+            
+            eintrag.marke = tbMarke.Text;
+            eintrag.modell = tbModell.Text;
```

Comment:
Als Error-Handling würde ich wahrscheinlich einfach alle Zuweisungen, bei denen geparsed wird, in ein try/catch packen und mit einer MessageBox oder so bei Fehlern warnen